contracts/src/v0.8/operatorforwarder/dev/AuthorizedForwarder.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {ConfirmedOwnerWithProposal} from "../../shared/access/ConfirmedOwnerWithProposal.sol";
+import {AuthorizedReceiver} from "./AuthorizedReceiver.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+// solhint-disable custom-errors
+contract AuthorizedForwarder is ConfirmedOwnerWithProposal, AuthorizedReceiver {
+  using Address for address;
+
+  // solhint-disable-next-line chainlink-solidity/prefix-immutable-variables-with-i
+  address public immutable getChainlinkToken;
```

Comment:
this name is confusing, we normally don't name variables with `get`. Does this need to be maintained due to legacy?